src/engraving/libmscore/staff.cpp

```diff
@@ -720,8 +710,8 @@ Fraction Staff::currentKeyTick(const Fraction& tick) const
 
 void Staff::write(XmlWriter& xml) const
 {
-    int idx = this->idx();
-    xml.stag(this, QString("id=\"%1\"").arg(idx + 1));
+    xml.stag(this, QString("id=\"%1\"").arg(idx() + 1));
```

Comment:
I know it was like this already but it seems odd that we use `idx() + 1` as the id here, when there's already an _id property?

Reply:
This requires a lot of work. I'll change this in a separate PR